docs/Game/Unreal/setting-up.md

```diff
@@ -7,72 +7,71 @@ Let's get started!
 
-#### Generate a Visual Studio Project
+## Generate a Visual Studio Project
 
-1. Generate a Visual Studio Project by Right-Clicking '[ProjectName].uproject' and selecting 'Generate Visual Studio project files.'
+1. Generate a **Visual Studio Project** by Right-Clicking **'[ProjectName].uproject'** and selecting **Generate Visual Studio project files**
 
-2. Open the generated 'Visual Studio solution file (.sln)' and check if the plugin in included in the solution explorer.
+2. Open the generated **Visual Studio solution file (.sln)** and check if the plugin in included in the solution explorer.
 
-3. Locate *GameInstance.h* if already created. If not Add C++ class from **Content Browser**, check 'Show All Classes' and select 'GameInstance'. Name your class 'MyGameInstance'.
+3. Locate **GameInstance.h** if already created. If not Add C++ class from **Content Browser**, check **Show All Classes** and select **GameInstance**. Name your class **'MyGameInstance'**.
+Open **MyGameInstance.h** and include the following code:
 
-Open `MyGameInstance.h` and include the following code:
+   ```cpp
+      #include "AnkrClient.h"
 
-```c++
-   #include "MirageClient.h"
+      UPROPERTY()
+      UAnkrClient* ankrClient;
 
-   UPROPERTY()
-   UMirageClient* mirageClient;
+      UFUNCTION(BlueprintCallable, Category = "ANKR SDK")
+      UAnkrClient* GetAnkrClient();
+   ```
 
-   UFUNCTION(BlueprintCallable, Category = "ANKR SDK")
-   UMirageClient* GetMirageClient();
-```
-
-4. Open `MyGameInstance.cpp` and include the following code:
+4. Open **MyGameInstance.cpp** and include the following code:
 
-```c++
-UMirageClient* UMyGameInstance::GetMirageClient()
-{
-	if (mirageClient == nullptr)
-	{
-		mirageClient = NewObject<UMirageClient>();
-	}
+   ```cpp
+   UAnkrClient* UMyGameInstance::GetAnkrClient()
+   {
+      if (ankrClient == nullptr)
+      {
+         ankrClient = NewObject<UAnkrClient>();
+      }
 
-	return mirageClient;
-}
-```
+      return ankrClient;
+   }
+   ```
 
-#### Add AnkrSDK to Build.cs
+## Add AnkrSDK to Build.cs
 
-Add "AnkrSDK" to your Unreal Project/Source/Unreal Project/Build.cs as shown below:
+Add **AnkrSDK** to your Unreal Project/Source/Unreal Project/Build.cs as shown below:
 
-```c++
+```cpp
 PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "AnkrSDK" });
 ```
 
-Click ***Edit->Project Settings->Maps and Modes*** and select your newly created or already created 'GameInstance' from the 'GameInstance Class' dropdown.
+Click **Edit > Project Settings >Maps and Modes** and select your newly created or already created **'GameInstance'** from the **'GameInstance Class'** dropdown.
 
-Now you can call all the functions in blueprint by getting 'GetGameInstance->GetMirageClient'.
+Now you can call all the functions in blueprint by getting **'GetGameInstance >GetAnkrClient'**.
```

Comment:
**'GetGameInstance >GetAnkrClient'**. > **'GetGameInstance > GetAnkrClient'**.

Reply:
space is missing

Reply:
done